server/team/team.contoller.js

```diff
@@ -95,5 +95,60 @@ function deleteTeam(req, res) {
     });
   });
 }
+/*
+vote: {
+  canVote: Boolean,
+  votesFor: [String],
+  receivedVotes: [String]
+}
+*/
 
-module.exports = { create, update, list, single, deleteTeam };
+function vote(req, res) {
+  const voteKey = (req.params.isAdd ? '$set' : '$pull');
```

Comment:
Because pushing new `teamId`s to the vote.votesFor array is not reliable, we have to send the array of three current votes for each team when the radio button is clicked. It is not coming over to the server correctly. I will point out the origin in the view.